dts/bindings/spi/spi.yaml

```diff
@@ -0,0 +1,31 @@
+---
+title: SPI Base Structure
+id: spi
+version: 0.1
+
+description: >
+    This binding gives the base structures for all SPI devices
+
+child:
+    bus: i2c
```

Comment:
What does this mean? Why are SPI and I2C in a parent/child relationship here?

Reply:
I used i2c.yaml as a template and I didn't remove it. Fixed.